loopchain/channel/channel_service.py

```diff
@@ -280,13 +280,17 @@ async def __init_peer_auth(self):
             utils.exit_and_msg(f"peer auth init fail cause : {e}")
 
     def __init_block_manager(self):
-        logging.debug(f"__load_block_manager_each channel({ChannelProperty().name})")
+        logging.debug(f"__init_block_manager() : channel({ChannelProperty().name})")
+
+        channel_name = ChannelProperty().name
+        develop = command_arguments.command_values.get(command_arguments.Type.Develop, False)
+        store_id = f"{ChannelProperty().peer_port}_{channel_name}" if develop else channel_name
```

Comment:
Just question.
What is store_id?
It seems that the format of old_store_id is different from that of store_id.

Which one is the correct format of store_id?
1. f"{peer_target}"
2. f"{peer_target}_{channel_name}"
3. f"{peer_port}_{channel_name}

Reply:
It's a just name to distinguish path of key-value store.
The name depends on the case of `develop` or `product`.
It need several paths in `develop`, but one path in `product`.
- `{peer_port}_{channel_name}` in develop
- `{channel_name}` in product

`{peer_target}_{channel_name}` is old one `ip:port` included in dbpath.